src/main/scala/firrtl/transforms/formal/RemoveVerificationStatements.scala

```diff
@@ -19,7 +19,7 @@ class RemoveVerificationStatements extends Transform
   with PreservesAll[Transform] {
 
   override def prerequisites: Seq[TransformDependency] = Seq.empty
-  override def optionalPrerequisites: Seq[TransformDependency] = Seq.empty
+  override def optionalPrerequisites: Seq[TransformDependency] = Seq(Dependency(ConvertAsserts))
```

Comment:
What's your thinking here for specifying both `optionalPrerequisiteOf` in `ConvertAsserts` and an `optionalPrerequistes` here?

I'm not opposed to it, just wondering if you've identified some benefit/good pattern as I would think one or the other is sufficient.

Reply:
I actually am generally not in favor of this pattern (I would prefer to just use `optionalPrerequisites`), but the "two-sided" listing seems to have been adopted for other similar recent additions, so I was doing it for what I thought was maybe consistency.

Reply:
I agree. I'm in favor of just an `optionalPrerequisites` specification. No harm done and it's consistent with existing work. Fine the way it is, then.

Auditing the transforms to remove dependency over-specification can be done at a later point. Part of that might be good to come up with some "style guide" for transform writers. Future work...